XPlotter.cpp

```diff
@@ -392,9 +392,6 @@ int main(int argc, char* argv[])
 	// check free RAM
 	if (freeRAM < nonces_per_thread * threads * PLOT_SIZE * 2) nonces_per_thread = freeRAM / threads / PLOT_SIZE / 2;
 
-	//ajusting
-	nonces_per_thread = (nonces_per_thread / (bytesPerSector / SCOOP_SIZE)) * (bytesPerSector / SCOOP_SIZE);

```

Comment:
If we set:
n = nonces_per_thread 
b = bytesPerSector 
s = SCOOP_SIZE

line 396 can be re-written as:
n = (n / (b / s)) * (b / s);

if we then set:
bds = b/s

then we get: 
n = (n / bds) * (bds);

so isn't the division redundant then?